examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -169,6 +169,8 @@ def __post_init__(self):
 
 class TrainState(train_state.TrainState):
     dropout_rng: jnp.ndarray
+    grad_accum: jnp.ndarray
+    optimizer_step: int
```

Comment:
Would it maybe to just use `state.step` instead of creating a new `optimizer_step`?